dist/app.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
the directory ```dist``` should not appear in the commit. I know that it's just for purposes of deployment. But anyway. Try please to avoid this practice.